tests/test_security_http_digest_optional.py

```diff
@@ -66,5 +66,5 @@ def test_security_http_digest_incorrect_scheme_credentials():
     response = client.get(
         "/users/me", headers={"Authorization": "Other invalidauthorization"}
     )
-    assert response.status_code == 403, response.text
```

Comment:
I think exception raising should also be tested

Reply:
This is the "optional" test suite, the exception is tested here: https://github.com/arthurio/fastapi/blob/digest-except/tests/test_security_http_digest.py

Reply:
@juntatalor Is there anything else you need to move forward?